orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/cuppa/CuppaData.java

```diff
@@ -17,12 +17,19 @@ public interface CuppaData
     @NotNull
     List<CuppaPrediction> predictions();
 
-    int simpleDups32To200B();
```

Comment:
Oh, I see this was later added back in. I'm glad you found these were still available in v2 @luan-n-nguyen, having these again removes one of my worries for this upgrade.

Though is the change of type from int to Integer necessary? How would that affect compat?

Reply:
I made these Integers so that they are nullable so that `doNotCrashOnMissingEntries` passes in [CuppaDataFactoryTest](https://github.com/hartwigmedical/hmftools/blob/e4ec8441ae4f8e690639cd76e55185f9f1afcff1/orange/src/test/java/com/hartwig/hmftools/orange/algo/cuppa/CuppaDataFactoryTest.java#L143C19-L143C19)

Reply:
Could you clarify this a bit more? That test looks like it was originally added several months ago (though modified recently), and they couldn't be null previously. Does the cuppa v2 make it possible for these being null in some cases? If not, something seems suspicious about that test case.

Reply:
This change is not backwards compatible, while it's an improvement, old clients could now fail with an NPE as they'd expect the magic -1.  We need to revert it back to @NotNull/primitive.

Reply:
I've now made the SV feature values be primitive ints again, and have them be -1 if they are "null"